dietpi/dietpi-autostart

```diff
@@ -90,6 +90,8 @@ _EOF_
 			"3" "RetroPie"
 			"4" "OpenTyrian"
 			"5" "DietPi-Cloudshell"
+			"6" "MATE: Desktop"
+			"7" "GNUStep: Desktop"
```

Comment:
You'll also need to add code to run these desktops in the login script: https://github.com/Fourdee/DietPi/blob/testing/dietpi/login#L65-L101